src/layers/point-layer/point-layer.js

```diff
@@ -297,6 +319,27 @@ export default class PointLayer extends Layer {
     this.updateMeta({bounds});
   }
 
+  getTextOffset(config, radiusScale, getRadius, mapState) {
+    const distanceScale = getDistanceScales(mapState);
+    const xMult = config.anchor === 'middle' ? 0 :
+      config.anchor === 'start' ? 1 : -1;
+    const yMult = config.alignment === 'center' ? 0 :
+      config.alignment === 'bottom' ?  1 : -1;
+
+    const sizeOffset = config.alignment === 'center' ? 0 :
+    config.alignment === 'bottom' ?  config.size : config.size;
```

Comment:
add space at the beginning of the statement to be consistent with the rest of the method

Reply:
done